skyplane/planner/planner.py

```diff
@@ -240,13 +240,19 @@ def plan(self, jobs: List[TransferJob]) -> TopologyPlan:
             mux_or = src_program.add_operator(GatewayMuxOr(), parent_handle=obj_store_read, partition_id=partition_id)
             for i in range(n_instances):
                 src_program.add_operator(
-                    GatewaySend(target_gateway_id=dst_gateways[i].gateway_id, region=src_region_tag, num_connections=self.n_connections),
+                    GatewaySend(
```

Comment:
Could you modify all the `GatewaySend` and `GatewayReceive` operators in the other planners to also include compression and encryption?